### DISCUSSION TITLE
Encryption

### DISCUSSION THREAD

**Discussion by gsc29** [olahallengren/sql-server-maintenance-solution#661] [General]
Hi,

Simple question. Do your scripts deal with encrypting data for backups?  I can see there is an option but couldn't find documentation on its use.

Thanks

**Comment by pbarryuk** [comment]
It’s covered in the docs for the back stored proc - [https://ola.hallengren.com/sql-server-backup.html](https://ola.hallengren.com/sql-server-backup.html)
